packages/flare/bin/cron_job_ingest_events.py

```diff
@@ -62,6 +63,11 @@ def main(logger: Logger, app: client.Application) -> None:
     logger.info(f"Retrieved {events_retrieved_count} events")
 
 
+def create_flare_index(service: Service) -> None:
+    if APP_NAME not in service.indexes:
```

Comment:
we should probably make sure we only create the index once since they might decide to use their own index and delete ours. Could we add a flag in the Collection to say that we did it once?